tools/wpt/virtualenv.py

```diff
@@ -121,17 +121,19 @@ def install(self, *requirements):
         # occurs while packages are in the process of being published.
         call(self.pip_path, "install", "--prefer-binary", *requirements)
 
-    def install_requirements(self, requirements_path):
-        with open(requirements_path) as f:
-            try:
-                self.working_set.require(f.read())
-            except Exception:
-                pass
-            else:
-                return
-
-        # `--prefer-binary` guards against race conditions when installation
-        # occurs while packages are in the process of being published.
-        call(
-            self.pip_path, "install", "--prefer-binary", "-r", requirements_path
-        )
+    def install_requirements(self, *requirements_paths):
+        install = []
```

Comment:
This isn't new, but can you add a comment explaining why we check which things need to be installed and not? Presumably a call to pip install is slow even when all dependencies are up to date?